### ISSUE TITLE
How to call response_hooks with auto-instrumentation?

### ISSUE THREAD

**Issue by rdpravin1895** [open-telemetry/opentelemetry-python-contrib#1020]
Is there a way to set response_hooks for instrumentors during auto-instrumentation? Currently, I am setting it this way for django manual instrumentation:

```
DjangoInstrumentor().instrument(response_hook=response_hook)
```

But how can I achieve the same with auto-instrumentation or sitecustomize?

**Comment by ocelotl** [comment]
I don't think it is possible right now, it would be necessary to implement a way of getting the response hook somehow from environment variables that can be set prior to auto instrumentation execution. I am making this issue a feature request to track that.
